client/src/InfoBase/index_data.js

```diff
@@ -136,15 +136,19 @@ const index_lang_lookups = {
     en: "index-basic-eng.html",
     fr: "index-basic-fra.html",
   },
+  a11y_version_title: {
+    en: "Text-only version",
+    fr: "Version texte seulement",
```

Comment:
Actually I think this should be "version texte" as well, because otherwise the full phrase is "Version texte seulement, 
 recommandé pour les utilisateurs de lecteur d'écran" which looks like the comma is an error and you're saying that the text version is only recommended for screen reader users.